docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -278,13 +278,14 @@ Yes, if action is `edit`.
 
 [width="100%",options="header"]
 |==============================================
-|`type` field |`value` field |Description
-| `add_tags` | String[] |  Add tags to rules
-| `delete_tags` | String[] |  Delete tags from rules
-| `set_tags` | String[] |  Overwrite tags with value(s) in rules
-| `add_index` | String[] |  Add index patterns to rules
-| `delete_index` | String[] |  Delete index patterns from rules
-| `set_index` | String[] |  Overwrite index patterns with value in rules
+| `type` field | `value` field | Description
+| `add_tags` | String[] | Add tags to rules
+| `delete_tags` | String[] | Delete tags from rules
+| `set_tags` | String[] | Overwrite tags with value(s) in rules
+| `add_index_patterns` | String[] | Add index patterns to rules
+| `delete_index_patterns` | String[] | Delete index patterns from rules
+| `set_index_patterns` | String[] | Overwrite index patterns with value in rules
+| `set_timeline` | { timeline_id: String; timeline_title: String } | Overwrite timeline template in rules
```

Comment:
These nested field names should probably use code format too? And "Timeline" is capitalized.
```suggestion
| `set_timeline` | { `timeline_id`: String; `timeline_title`: String } | Overwrite Timeline template in rules
```